src/components/chatbox/message.module.css

```diff
@@ -0,0 +1,94 @@
+.message {
+	display: flex;
+	flex-direction: row;
+	gap: 12px;
+	padding: 20px 16px;
+	background: var(--token-color-palette-neutral-0);
+}
+
+.user {
+	border-bottom: 1px solid var(--token-color-palette-neutral-300);
+}
+
+.avatar,
+.avatar img {
+	width: 40px;
+	height: 40px;
+	border-radius: 6px;
+}
+
+.content {
+	padding-top: 10px; /* simulate vertically centering the first line with avatar/icon */
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+}
+
+.userInput {
+	color: var(--token-color-foreground-primary);
+	white-space: pre-line; /* visually respect user-inputted line breaks */
+}
+
+.gutter {
+	min-width: 40px;
+}
+
+.assistant {
+	background: var(--token-color-surface-strong);
+}
+
+.icon {
+	--icon-color: var(--token-color-foreground-highlight-on-surface);
+}
+
+.markdown {
+	color: var(--token-color-foreground-primary);
+	& pre {
+		overflow-x: auto;
+		white-space: pre-wrap;
+		word-wrap: break-word;
+	}
+}
+
+.assistantMessageFooter {
+	width: 100%;
+	padding: 24px 0 0;
+}
+
+.assistantMessageFooterHidden {
+	display: none;
+}
+
+.divider {
+	display: block;
+	width: 100%;
+	border-bottom: 1px solid var(--token-color-palette-neutral-300);
+	margin-bottom: 24px;
+}
+
+.actionButtons {
+	display: flex;
+	flex-direction: row;
+	gap: 16px;
+}
+
+.ratingLike {
```

Comment:
Mentioned in the relevant thread, and this might be more a design question than anything else, but I think we could resolve the `!important` here by using `secondary` state for the initial rating button state (same as is now), and then `primary` state for the "selected" rating state (then no need for color overrides).

<img width="1061" alt="CleanShot 2023-09-21 at 16 02 05@2x" src="https://github.com/hashicorp/dev-portal/assets/4624598/72425a6e-8c4b-48c4-9b7c-92a74f8551e1">

Reply:
📌 Oooh, related future thought, and likely outside the scope of this PR: should we consider consistency with the tutorials feedback form? cc @kellyschummer would love your take on this 🙇‍♂️

<img width="767" alt="CleanShot 2023-09-21 at 15 40 50@2x" src="https://github.com/hashicorp/dev-portal/assets/4624598/0b6d096e-c964-42d7-9b5c-fcec3115bb27">

<img width="728" alt="CleanShot 2023-09-21 at 15 41 30@2x" src="https://github.com/hashicorp/dev-portal/assets/4624598/e03d48ce-4f77-4dd0-9b12-ca492be2e51c">